bindings/Python/py11File.h

```diff
@@ -24,6 +24,8 @@ namespace py11
 class File
 {
 public:
+    static std::string ReadOrder;
```

Comment:
`m_ReadOrder`

Reply:
Not sure about the need for `static`, a single workflow can be using multiple instances of File with different data layout order. Making it static is basically a global across all files.

Reply:
`ReadOrder` is just the default order, and I deliberately wanted it to be "global", so that one could do
```python
import adios2
adios2.File.read_order = 'C'
```
to restore the old behavior. This wouldn't work with a per-instance member variable.

I can prefix it with `m_`, i wasn't sure you also do that for static class variables.

Reply:
@germasch not sure I am following. 

```
import adios2
adios2.File.read_order = 'C'

f1 = adios2.open("F.bp")
cm = f1.read(...)
f2 = adios2.open("C.bp")
rm = f2.read(...)
```
would this work? Is there a test for this since it's a new feature?